smoke-tests/src/test/java/com/splunk/opentelemetry/AppServerTest.java

```diff
@@ -44,38 +46,38 @@ protected void assertWebAppTrace(ExpectedServerAttributes serverAttributes, Test
     String url = getUrl("/app/greeting", false);
 
     Request request = new Request.Builder().get().url(url).build();
-    String responseBody = tryGetResponse(request);
+    String responseHeadersAndBody = tryGetResponse(request);
 
     TraceInspector traces = waitForTraces();
 
     Set<String> traceIds = traces.getTraceIds();
 
-    Assertions.assertEquals(1, traceIds.size(), "There is one trace");
+    assertEquals(1, traceIds.size(), "There is one trace");
```

Comment:
Should this be changed as well?

Reply:
`traceIds` here is `Set`, so it's size should be correct.